<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Рисовалка</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="maingridLayout" columnstretch="0">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>1</string>
       </attribute>
       <widget class="myGridWidget" name="gridWidget_1" native="true">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>876</width>
          <height>513</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_1" rowstretch="1,1,1" columnstretch="1,3,1">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <property name="spacing">
          <number>20</number>
         </property>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_14">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_11">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_13">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_12">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="GraphicsWidget" name="graphics1" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>2</string>
       </attribute>
       <widget class="myGridWidget" name="gridWidget_2" native="true">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>876</width>
          <height>513</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,1" columnstretch="1,3,1">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <property name="spacing">
          <number>20</number>
         </property>
         <item row="1" column="1">
          <widget class="GraphicsWidget" name="graphics2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_23">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_21">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_22">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_24">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>3</string>
       </attribute>
       <widget class="myGridWidget" name="gridWidget_3" native="true">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>876</width>
          <height>513</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,1,1" columnstretch="1,3,1">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <property name="spacing">
          <number>20</number>
         </property>
         <item row="1" column="1">
          <widget class="GraphicsWidget" name="graphics3" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_34">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_33">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_31">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_32">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLeft"/>
   <addaction name="actionExit"/>
   <addaction name="actionRight"/>
   <addaction name="actionLines"/>
   <addaction name="actionRandom"/>
  </widget>
  <action name="actionLeft">
   <property name="text">
    <string>&lt;-</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>X</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>-&gt;</string>
   </property>
  </action>
  <action name="actionLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>#</string>
   </property>
  </action>
  <action name="actionRandom">
   <property name="text">
    <string>Перестановка</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsWidget</class>
   <extends>QWidget</extends>
   <header>graphicswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>myGridWidget</class>
   <extends>QWidget</extends>
   <header>mygridwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
